crates/uv-python/src/version_files.rs

```diff
@@ -111,20 +124,11 @@ impl PythonVersionFile {
         Self::try_from_path(path).await
     }
 
-    pub async fn discover_user_config(
-        user_config_working_directory: impl AsRef<Path>,
-        options: &DiscoveryOptions<'_>,
-    ) -> Result<Option<Self>, std::io::Error> {
-        if !options.no_config {
```

Comment:
We'll respect the `no_config` logic above instead, which has a message for when it's applied.